themes/af-magic.zsh-theme

```diff
@@ -6,7 +6,7 @@ if [ $UID -eq 0 ]; then NCOLOR="red"; else NCOLOR="green"; fi
 local return_code="%(?..%{$fg[red]%}%? ↵%{$reset_color%})"
 
 # primary prompt
-PROMPT='$FG[237]${(l.COLUMNS..-.)}%{$reset_color%}
+PROMPT='$FG[237]${(l.COLUMNS-1..-.)}%{$reset_color%}
```

Comment:
Can you try this instead?
```suggestion
PROMPT='%{$FG[237]%}${(l.COLUMNS..-.)}%{$reset_color%}
```

Reply:
I tried it and it unfortunately doesn't fix the issue.